support/routes/v2/isobot/ops/botService.ts

```diff
@@ -66,21 +39,13 @@ class BotService {
   }
 
   private checkA(domain: string) {
-    return ResultAsync.fromPromise(
-      Promise.race([
-        dns.resolve4(domain),
-        new Promise<null>((_, reject) =>
-          setTimeout(() => reject(null), DNS_TIMEOUT)
-        ),
-      ]),
-      () => {
-        logger.info({
-          message: "Error resolving A record",
-          meta: { domain, method: "checkA" },
-        })
-        return new Error()
-      }
-    )
+    return ResultAsync.fromPromise(dns.resolve4(domain), () => {
+      logger.info({
```

Comment:
message says error but the log level is info?

Reply:
sure, i'll mkae the change